src/Standards/PSR12/Docs/ControlStructures/ControlStructureSpacingStandard.xml

```diff
@@ -0,0 +1,124 @@
+<documentation title="Control Structure Spacing">
+    <standard>
+    <![CDATA[
+    The control structures must have one space after the control structure keyword, no spaces after the opening parenthesis and before the closing parenthesis, and one space between the closing parenthesis and the opening brace.
+    ]]>
+    </standard>
+    <code_comparison>
+        <code title="Valid: No space after the opening parenthesis in a single-line condition.">
+        <![CDATA[
+if <em>(</em>$expr) {
+}
+        ]]>
+        </code>
+        <code title="Invalid: Space after the opening parenthesis in a single-line condition.">
+        <![CDATA[
+if <em>( </em>$expr) {
```

Comment:
Should the "no space after the keyword" rule get its own code sample ? Or could this be included in this invalid code sample ?

```suggestion
if<em> ( </em>$expr) {
```

Reply:
Technically, this sniff doesn't check for the space after the keyword. This is covered by the `Squiz.ControlStructures.ControlSignature.SpaceAfterKeyword` sniff, so I'm not sure if we should include it in the documentation for this sniff.

I'm all for removing squiz rules from the PSR12 ruleset, and just modifying the `ControlStructuresSpacing` sniff to include all the checks as in the PSR-12 documentation for control structures, but I think that would be something for PHPCS v4.

Reply:
You are 100% correct! I thought this sniff was checking that too, but you are right, it does not. The updated standards description from commit a1795c1eebaf0063f4a70185eba6f2e2b8a31f40 confused me as that _does_mention the space after the control structure keyword.

In other words: No, there should not be a code sample for space after keyword, but the first `<standard>` block will need to be updated to remove the mention of the spacing after the keyword.

Reply:
And this bit _", and one space between the closing parenthesis and the opening brace"_ will also need to be removed as that is also not checked via this sniff.

Reply:
Yes, you're right about these. Will fix them now 👍🏼